docs/modules/usage/architecture/runtime.md

```diff
@@ -108,7 +107,7 @@ The hash-based tag ensures exact reproducibility, while the generic tag provides
      Example: `runtime:od_v0.8.3_ubuntu_tag_22.04`
 
 2. Build Process:
-   - a. Convert the base image name to an OD runtime image name.
+   - a. Convert the base image name to an OH Runtime image name.
       Example: `ubuntu:22.04` -> `runtime:od_v0.8.3_ubuntu_tag_22.04`
```

Comment:
There are a few "od" references in this mermaid graph 👀

Reply:
Yes, they need to stay for now - tech reasons 😁

Reply:
Or did we decide to keep it od_?
Cause there's a few "od_" references in this doc in case we decided to change those.

Reply:
There are still images with that prefix, will be fixed in another PR by Robert, afaik.